cmd/crictl/container.go

```diff
@@ -169,12 +169,50 @@ var runPullFlags = []cli.Flag{
 	},
 }
 
-var createContainerCommand = &cli.Command{
-	Name:      "create",
-	Usage:     "Create a new container",
-	ArgsUsage: "POD container-config.[json|yaml] pod-config.[json|yaml]",
-	Flags:     createPullFlags,
+var subcommands = []*cli.Command{{
+	Name:    "jsonschema",
+	Aliases: []string{"js"},
+	Usage:   "Display the JSON schema for the pod or container config",
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "pod",
+			Aliases: []string{"p"},
+			Usage:   "Print the pod JSON schema, which will be generated from the PodSandboxConfig of the CRI API",
+		},
+		&cli.BoolFlag{
+			Name:    "container",
+			Aliases: []string{"c"},
+			Usage:   "Print the container JSON schema, which will be generated from the ContainerConfig of the CRI API",
+		},
+	},
+	UseShortOptionHandling: true,
+	Action: func(c *cli.Context) error {
+		if !c.IsSet("pod") && !c.IsSet("container") {
+			return cli.ShowSubcommandHelp(c)
+		}
+
+		if c.IsSet("pod") {
+			if err := printJSONSchema(&pb.PodSandboxConfig{}); err != nil {
```

Comment:
this will print the schema of the version that crictl was compiled with, not the runtime version, correct?

Reply:
Yes, that's true.

Reply:
I wonder if we need to mention it at least in docs.

Reply:
Yep, done